mobile/test/swift/integration/IdleTimeoutTest.swift

```diff
@@ -157,27 +85,30 @@ static_resources:
 
     let client = engine.streamClient()
 
-    let requestHeaders = RequestHeadersBuilder(method: .get, scheme: "https",
-                                               authority: "example.com", path: "/test")
-      .build()
+    let port = String(EnvoyTestServer.getEnvoyPort())
+    let requestHeaders = RequestHeadersBuilder(
+      method: .get, scheme: "http", authority: "localhost:" + port, path: "/"
+    )
+    .build()
 
     client
       .newStreamPrototype()
       .setOnError { error, _ in
-        XCTAssertEqual(error.errorCode, 0)
+        XCTAssertEqual(error.errorCode, 4)
```

Comment:
Apparently I'm testing a different timeout for 4 is ENVOY_REQUEST_TIMEOUT which is an idle timeout on the request path, triggerd by sending no end stream.